.env.beta

```diff
@@ -1,5 +1,6 @@
 OPTIC_RUST_DIFF_ENGINE=true
 REACT_APP_LEARN_API_MODE=true
 REACT_APP_DIFF_REVIEW_PAGE=true
+GITFLOW_CAPTURE=true
```

Comment:
We'll want to prefix env variables solely for Optic's use, as ENV variables are like global variables.